config/rbac/role_labels_patch.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
why this line?

Reply:
signal the start of the yaml

Reply:
since this is not a multi-doc yaml, I don't think it's necessary

Reply:
done